Library/Homebrew/cmd/install.rb

```diff
@@ -142,7 +145,12 @@ def check_cellar
   def perform_preinstall_checks
     check_ppc
     check_writable_install_location
-    check_xcode
+    if MacOS::Xcode.installed?
+      check_xcode
+    else
+      opoo "You have not installed Xcode."
+      puts "Bottles may install correctly, but builds will fail!"
```

Comment:
Alright! Would checking the formula's dependency tree for 100% bottled-ness and failing if not be a good solution?